app/views/blocks/graduationDateQuestion.html

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-sm-4 stacked-spacing-col-sm">
+    <crs-datetimepicker
+      model="answer.value"
+      graduation-date="true"
```

Comment:
As this is very different to all the other dates. I wonder if we need to make the date modal appear on focus as the user might be confused about what to type.